Sources/Specs/GameSpec.swift

```diff
@@ -0,0 +1,18 @@
+import Dictionary
+import SnowManLogic
+import Swiftest
+
+class GameSpec: Swiftest.Spec {
+  	let spec = describe("game logic") {
+  		let dictionary = DictionaryManager()
+		let game = Game(dictionary: dictionary)
```

Comment:
these don't need to be abstracted into variables at this scope yet, they can exist inside of the `it` block.